common/flatpak-transaction.c

```diff
@@ -2200,7 +2200,8 @@ search_for_dependency (FlatpakTransaction  *self,
       if (state == NULL)
         {
           g_debug ("Can't get state for remote %s: %s", remote, local_error->message);
-          return FALSE;
+          g_propagate_error (error, g_steal_pointer (&local_error));
+          return NULL;
```

Comment:
I think this unnecessarily makes this a hard error. We're doing this on all remotes, looking for a possible runtime. If we find one in runtime a, then we should not fail the entire install because a network issue in runtime b.

I think the return should be a continue instread.

Reply:
:+1: Done